PhysicsTools/Heppy/python/analyzers/objects/LeptonAnalyzer.py

```diff
@@ -200,6 +200,7 @@ def makeLeptons(self, event):
         if self.doIsoAnnulus:
             for lep in event.inclusiveLeptons:
                 self.attachIsoAnnulus04(lep)
+                self.attachIsoAnnealusMiniIso(lep)
```

Comment:
why one is Annulus and the other Annealus?
could we use just one spelling, whatever is more correct

Reply:
Yes, going to correct that asap in the different releases